README.md

```diff
@@ -3,8 +3,9 @@
 [![Build Status](https://travis-ci.org/logzio/sawmill.svg?branch=master)](https://travis-ci.org/logzio/sawmill)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.logz.sawmill/sawmill/badge.svg)](http://mvnrepository.com/artifact/io.logz.sawmill/sawmill)
 
-*Update: April 22, 2020*
-__Please note: there is currently a known build error due to our dependency on MaxMind GEOIP changes. We are working to resolve this by May, 15th 2020. Please contact for additional information__
+*Update: June 25, 2020*
+We introduced a new major version 2.0.0 with breaking changes that affect *GeoIpProcessor* usage due to MaxMind license policy changes.
```

Comment:
I think the phrasing should be a little different:

The 2.0 release of Sawmill introduces a breaking change to the *GeoIpProcessor* to comply with the updated license of the MaxMind Lite database.